CoreScriptsRoot/CoreScripts/GamepadMenu.lua

```diff
@@ -667,9 +669,16 @@ local function setupGamepadControls()
 			setButtonEnabled(button, enabled)
 		end
 	end
+	
+	local function enableRadialMenu()
+		ContextActionService:BindCoreAction(toggleMenuActionName, doGamepadMenuButton, false, Enum.KeyCode.ButtonStart)
+	end
+	
+	local loadedConnection = game.Loaded:connect(enableRadialMenu)
 	StarterGui.CoreGuiChangedSignal:connect(setRadialButtonEnabled)
-
-	ContextActionService:BindCoreAction(toggleMenuActionName, doGamepadMenuButton, false, Enum.KeyCode.ButtonStart)
+	
+	game.Close:wait()
+	loadedConnection:disconnect()
```

Comment:
Couldn't you put this in enableRadialMenu? Not a big deal but it's a little cleaner and you don't need the game.Close:wait(). You'd have to predefine loadedConnection before defining enableRadialMenu though.